js/viz/gauges/utils/resolve_label_overlapping.js

```diff
@@ -0,0 +1,115 @@
+const ARC_COORD_PREC = 5;
+
+import { normalizeAngle, normalizeArcParams } from '../../core/utils';
+
+export function dividePoints(bars) {
+    return bars.reduce(function(stackBars, bar) {
+        const angle = normalizeAngle(bar._angle);
+        const isRightSide = angle <= 90 || angle >= 270;
+        bar._text._lastCoords = { x: 0, y: 0 };
+
+        (isRightSide ? stackBars.right : stackBars.left)
```

Comment:
let's move it to a variable `const barToExtend = isRightSide ? stackBars.right : stackBars.left;`

Reply:
fixed